cypress/e2e/cloud/about.test.ts

```diff
@@ -2,25 +2,25 @@ import {Organization} from '../../../src/types'
 
 // resolve flakey behavior for the feature flag setting
 describe.skip('About Page for free users with only 1 user', () => {
-  beforeEach(() => {
-    cy.flush()
-
-    cy.signin().then(() => {
-      cy.get('@org').then(({id}: Organization) => {
-        cy.setFeatureFlags({
-          uiUnificationFlag: true,
-        }).then(() => {
-          cy.quartzProvision({
-            accountType: 'free',
-            hasUsers: false,
+  beforeEach(() =>
+    cy.flush().then(() =>
+      cy.signin().then(() => {
```

Comment:
we must wait on the async `cy.flush()` command to complete before signing in. Had to change this in every beforeEach